### PR TITLE
pre-emptive 0.6 compatability fix

### PR DESCRIPTION
**Pull request by ZacLN**
Not urgent

### CODE REVIEW

**Comment by ararslan** [comment]
Since this package supports Julia 0.5, wherein `isa` isn't parsed as an infix operator, that conditional should probably also have a version check. That is, something like
```julia
!(¬t in ops || ¬t === :in || (VERSION >= v"0.6.0-dev.whatever" && ¬t === :isa))
```
It might work like you have it in 0.5 but I'm not sure.

Would also be good to add a test if possible.

**Comment by ZacLN** [comment]
Will do, how can i find the whatever in `v"0.6.0-dev.whatever` from the commit code?

**Comment by ararslan** [comment]
See https://github.com/JuliaLang/Compat.jl#developer-tips

**Comment by ZacLN** [comment]
Cheers!

File: src/parser.jl [hunk 1]
```diff
@@ -444,7 +444,7 @@ function parse_comparison(ps::ParseState, ts::TokenStream, ops=INEQ_OPS)
     isfirst = true
     while true
         t = peek_token(ps, ts)
-        if !(¬t in ops || ¬t === :in)
+        if !(¬t in ops || ¬t === :in || (VERSION >= v"0.6.0-dev.1471" && ¬t === :isa))
```
Comment (ararslan) [inline src/parser.jl]: Sorry, I guess the version should have a + instead of a ., i.e. `v"0.6.0-dev+1741"`. That was my bad.

Comment (ZacLN) [inline src/parser.jl]: to be fair there was an example right below for me...

**Comment by ice1000** [comment]
Any plan for merging this?

**Pull request closed without merging**